docs/code-examples/text_document.py

```diff
@@ -6,6 +6,9 @@
 
 rr.init("rerun_example_text_document", spawn=True)
 
-rr2.log("text_document", rr2.TextDocument(
-    body="Hello, TextDocument!",
-))
+rr2.log(
+    "text_document",
+    rr2.TextDocument(
+        body="Hello, TextDocument!",
+    ),
+)
```

Comment:
remove the commas so `black` formats it all as one line; this is pretty jarring as-is and will take a bunch of unneeded space in the docstring
```suggestion
rr2.log(
    "text_document",
    rr2.TextDocument(
        body="Hello, TextDocument!"
    )
)

Reply:
I had it all on one line until black formatted it like this for me